go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/actgardner/gogen-avro
```

Comment:
Can you exclude these from the PR? I haven't really caught up on the module system, I'd rather add that separately.

Reply:
I have removed the go.mod and go.sum files